### ISSUE TITLE
Buy me a coffee

### ISSUE THREAD

**Issue by gnadelwartz** [gnadelwartz/talk-with-me#3]
hi @smashah

I used buy me a coffee the first time,  hope you received your coffee because I stuck at the message processing payment and get no feedback, even I tried two times.

kay

**Comment by smashah** [comment]
@gnadelwartz thanks so much I super appreciate it!

**Comment by gnadelwartz** [comment]
yeah, now I can see the transaction in PayPal also, it worked in both try, so you have double caffeine now ... 😁
